### PR TITLE
Added timestamp code to overlay

### PR DESCRIPTION
**Pull request by changesbyjames**
## Describe your changes

Added a machine-readable time code on the overlay.

## Notes for testing your change

...

### CODE REVIEW

**Review by pjeweb — APPROVED** [review]

File: apps/website/src/pages/stream/overlay.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ import {
 } from "@/utils/datetime";
 
 import logoImage from "@/assets/logo.png";
+const colors = ["#7E7E7E", "#4E3029"];
```
[the remark this answers is not in the document]
Comment (pjeweb) [reply apps/website/src/pages/stream/overlay.tsx:14]: I mean these colors are not really part of the design and more chosen to have enough contrast for decoding while not sticking out too much on stream. I am fine with them being explicitly defined here?!

[the remark this answers is not in the document]
Comment (pjeweb) [reply apps/website/src/pages/stream/overlay.tsx:14]: (but plus 1 for the order)

[the remark this answers is not in the document]
Comment (changesbyjames) [reply apps/website/src/pages/stream/overlay.tsx]: I've updated that!

File: apps/website/src/pages/stream/overlay.tsx [hunk 2]
```diff
@@ -225,6 +240,18 @@ const OverlayPage: NextPage = () => {
           </div>
         </div>
       </Transition>
+
+      <div className="absolute bottom-0 right-0 grid grid-cols-12">
+        {time.code.map((bit, idx) => (
+          <div
+            key={bit + idx}
```
[the remark this answers is not in the document]
Comment (changesbyjames) [reply apps/website/src/pages/stream/overlay.tsx]: I've made it more explicit, hopefully that's better!

**Review by MattIPv4 — APPROVED** [review]

**Pull request merged**